ament_tools/verbs/build/cli.py

```diff
@@ -200,38 +222,39 @@ def print_topological_order(opts, packages):
 
 def iterate_packages(opts, packages, per_package_callback):
     start_with_found = not opts.start_with
+    opts.skip_packages = opts.skip_packages or []
     install_space_base = opts.install_space
     package_dict = dict([(path, package) for path, package, _ in packages])
     for (path, package, depends) in packages:
         if package.name == opts.start_with:
             start_with_found = True
-        if not start_with_found:
+        if not start_with_found or package.name in opts.skip_packages:
             print('# Skipping: %s' % package.name)
-            continue
```

Comment:
Why remove the `continue` here and intend all of the following code in an `else` block?